libev.spec

```diff
@@ -0,0 +1,84 @@
+Name:             libev
+Version:          4.33
+Release:          1
+Summary:          High-performance event loop/event model with lots of features
+
+License:          BSD or GPLv2+
+URL:              http://software.schmorp.de/pkg/libev.html
+Source0:          %{name}-%{version}.tar.gz
```

Comment:
Looks like `libev-4.20-Respect-the-CFLAGS-if-defined.patch` is not applied any more since 4.23.

Reply:
Good that you checked. Leftover also in fedora then. :)
Better not to add it here either.